### PR TITLE
LCAM-876: Fix Invalid Helm Identifiers

### PR DESCRIPTION
**Pull request by Mhowell494**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/LCAM-876)

- Fixed invalid Helm identifiers causing deployments to fail.

### CODE REVIEW

File: helm_deploy/laa-crown-court-proceeding/templates/_environment.tpl [hunk 1]
```diff
@@ -37,19 +37,19 @@ env:
...
+    value: {{ .Values.cloud_platform.aws.sqs.queue.hearingResulted }}
   - name: PROSECUTION_CONCLUDED_QUEUE
-    value: {{ .Values.cloud-platform.aws.sqs.queue.prosecutionConcluded.url }}
+    value: {{ .Values.cloud_platform.aws.sqs.queue.prosecutionConcluded.url }}
   - name: PROSECUTION_CONCLUDED_LISTENER_ENABLED
-    value: {{ .Values.cloud-platform.aws.sqs.queue.prosecutionConcluded.listenerEnabled }}
+    value: {{ .Values.cloud_platform.aws.sqs.queue.prosecutionConcluded.listenerEnabled }}
   - name: PROSECUTION_CONCLUDED_SCHEDULE_ENABLED
-      value: {{ .Values.cloud-platform.aws.sqs.queue.prosecutionConcluded.scheduleEnabled }}
+      value: {{ .Values.cloud_platform.aws.sqs.queue.prosecutionConcluded.scheduleEnabled }}
```
Comment (nigelpain) [inline helm_deploy/laa-crown-court-proceeding/templates/_environment.tpl:52]: Does the underscore work ok? I looked at the docs https://v2.helm.sh/docs/chart_best_practices/#naming-conventions to double check and they seem to suggest using camelCase for values.

File: helm_deploy/laa-crown-court-proceeding/templates/_environment.tpl [hunk 2]
```diff
@@ -37,19 +37,19 @@ env:
   - name: EVIDENCE_API_OAUTH_CLIENT_SECRET
     value: {{ .Values.evidenceApi.clientSecret }}
   - name: CLOUD_PLATFORM_QUEUE_REGION
-    value: {{ .Values.cloud-platform.aws.sqs.region }}
+    value: {{ .Values.cloud_platform.aws.sqs.region }}
   - name: CLOUD_PLATFORM_QUEUE_ACCESS_KEY
-    value: {{ .Values.cloud-platform.aws.sqs.accessKey }}
```
Comment (skasthuri36326) [inline helm_deploy/laa-crown-court-proceeding/templates/_environment.tpl:42]: [Helm best practices:](https://helm.sh/docs/chart_best_practices/values/) Variable names should begin with a lowercase letter, and words should be separated with camelcase:

Comment (Mhowell494) [reply helm_deploy/laa-crown-court-proceeding/templates/_environment.tpl:42]: Much prefer underscore case, but point taken!

File: helm_deploy/laa-crown-court-proceeding/templates/_environment.tpl [hunk 3]
```diff
@@ -36,20 +36,20 @@ env:
     value: {{ .Values.evidenceApi.clientId }}
   - name: EVIDENCE_API_OAUTH_CLIENT_SECRET
     value: {{ .Values.evidenceApi.clientSecret }}
-  - name: CLOUD_PLATFORM_QUEUE_REGION
-    value: {{ .Values.cloud-platform.aws.sqs.region }}
-  - name: CLOUD_PLATFORM_QUEUE_ACCESS_KEY
-    value: {{ .Values.cloud-platform.aws.sqs.accessKey }}
-  - name: CLOUD_PLATFORM_QUEUE_SECRET_KEY
-    value: {{ .Values.cloud-platform.aws.sqs.secretKey }}
+  - name: cloudPlatform_QUEUE_REGION
```
Comment (skasthuri36326) [inline helm_deploy/laa-crown-court-proceeding/templates/_environment.tpl:39]: name should be CLOUD_PLATFORM_QUEUE_REGION and should sync with the docker-compose.override.yml

**Review by skasthuri36326 — APPROVED** [review]

**Pull request merged**